development/contribute-to-docs.md

```diff
@@ -0,0 +1,147 @@
+Timescale documentation is hosted in a [GitHub repository][timescale-docs-github] 
+and is open for contribution from all community members. If you 
+find errors or would like to add content to our docs, this tutorial 
+will walk you through the process.
+
+# Pre-requisites
+In order to modify documentation, you should have a working knowledge 
+of [git][install-git] and [Markdown][markdown-tutorial]. You will 
+also need to create a GitHub account.
+
+Be sure to read the [Timescale docs contribution styleguide][timescale-docs-style]. 
+You’ll see information about how we refer to aspects of Timescale, 
+how we format our docs, and special Markdown tags available to 
+you as you author your contribution.
+
+Before we accept any contributions, Timescale contributors need to
+sign the [Contributor License Agreement][timescale-github-cla] (CLA).
```

Comment:
Don't know that you need this to be a link, since whomever is not going to be signing the CLA until they actually submit a PR